rmarkdowns/Week_3.Rmd

```diff
@@ -327,6 +327,9 @@ leaflet() %>%
 
 Explore the data available in OpenStreetMap for your chosen location. What data would be relevant to topics you want to focus on? Try to create a draft map or two for your target area and target topics. 
 
+# Possible Topics for Project
+Since I am looking at the island of Oahu and the fact that there is a major overcrowding problem within the prison system, what if I were to Identify the optimal locations for new prison facility based on population density, land availability, and proximity to essential services (e.g., courts, hospitals).  Or the other idea I had was to assess the accessibility and efficiency of public transportation systems in Oahu. 
```

Comment:
Both are great ideas! I think the prison idea is really interesting. I would start with that one and see what data you can find that would be useful. I think the land availability might be hard to find, but I haven't tried it, so who knows? You might also start by just looking at current prison locations and see how far they are from courts/hospitals, etc. I'm happy to help with the step of pulling the data into RStudio and turning it into map layers (and maybe adding some basic spatial analysis, like buffer zones).